lib/collection/proxy-config.js

```diff
@@ -52,6 +63,15 @@ _.inherit((
         ProxyConfig.super_.call(this, options);
 
         // Assign defaults before proceeding
+        _.assign(ProxyConfig.prototype, /** @lends ProxyConfig.prototype */ {
```

Comment:
Why are you doing this inside every constructor call? You need to do it just once, outside the constructor block.